spec/factories.rb

```diff
@@ -0,0 +1,19 @@
+FactoryBot.define do
```

Comment:
This is not wrong.
But I recommend you to create factory for each class.
Unless otherwise, the file can be wrong.
If you understand it and you do this just because there're small amount of data, it's fine.
You don't need to fix this this time, anyway.